### ISSUE TITLE
Unable to save data

### ISSUE THREAD

**Issue by namik-mesic** [timhul/ClassicSim#166]
Greetings,

Love the tool, it seems very useful. However it looks extremely difficult to set up every single time I open the tool. Is it intended to reset all entries after close? Or am I missing something? I haven't noticed any saving features.

**Comment by veluria** [comment]
Did you unpack the archive before running it? It should save your data automatically when closing the program.

**Comment by namik-mesic** [comment]
I did get a crash once before exiting normally, but after building my data anew I had no issues.

**Comment by veluria** [comment]
Yeah, when the program crashes, it will not save the data. So if you've made a lot of changes and want to be safe, you should probably close it once before running the sim.

**Comment by namik-mesic** [comment]
Cheers that helps.

Namik Mešić, MSc.
Direktor, InnoTech d.o.o.
+387 62 209 271 | namik.mesic@innotech.ba
innotech.ba | Skype: namik_mesic | Milana Preloga 1

________________________________

**Comment by timhul** [comment]
Hi,

As veluria said the application does not attempt to mitigate any crashes so changes are lost. I think we should probably save the setup whenever the user starts running the sim.
